### ISSUE TITLE
Enhance the bulk actions during peak traffic

### ISSUE THREAD

**Issue by pixernot** [KeypDemoCorp/awesome-ui#200]
Accessibility should be considered when implementing this.

**Comment by alachaum** [comment]
Let's not over-engineer this, the simple fix is fine for now.
